packages/e2e/lib/e2e.dart

```diff
@@ -29,6 +30,14 @@ class E2EWidgetsFlutterBinding extends LiveTestWidgetsFlutterBinding {
           if (!_allTestsPassed.isCompleted) {
             _allTestsPassed.complete(true);
           }
+
+          if (!webDriverActionPipe.isCompleted) {
+            webDriverActionPipe.complete(Future.value(WebDriverAction.noop()));
```

Comment:
Is the analyzer ok with this? I thought we had the lint turned ont hat would want this to be `Future<WebDriverAction>.value`.  Ditto below.

Reply:
done.